rpc/http.go

```diff
@@ -30,6 +30,8 @@ import (
 	"time"
 )
 
+type contextKey string
+
```

Comment:
Is it necessary to use a cname here

Reply:
yes,it's better not to use string as key in the context.withvalue() ,i thought

Reply:
OK, it seems from golint rules

Reply:
It will be better to use name of  ```httpContextKey or rpcContextKey``` with more meanings than ```contextKey``` here .

Reply:
ok,
it avoid key collisions by defining a separate type,although may never happen :)